python/ray/data/_internal/execution/resource_manager.py

```diff
@@ -117,6 +117,46 @@ def _estimate_object_store_memory(self, op, state) -> int:
 
         return mem_op_internal + mem_op_outputs
 
+    def _estimate_object_store_memory_per_node(
+        self, op: PhysicalOperator, state: "OpState"
+    ) -> Dict[str, int]:
+        usage_per_node = defaultdict(int)
+
+        # iterate through inqueue
+        for bundle in op._metrics._internal_inqueue._bundle_to_nodes:
+            for _, meta in bundle.blocks:
+                node_id = (
+                    meta.exec_stats.node_id
+                    if meta.exec_stats and meta.exec_stats.node_id
+                    else "UNKNOWN"
```

Comment:
use the constant. maybe just move the entire function to `op_runtime_metrics.py`